doc/source/changes/version_0_22.rst.inc

```diff
@@ -151,42 +159,48 @@ Miscellaneous improvements
     >>> test.i[1].split('-')
     ['a1', 'a2']
 
+* updated AxisCollection.replace so as to replace one, several or all axes.
+
 * made create_sequential axis argument accept all the ways to create axes supported by other array creation functions
   like, for example, using a string definition (closes :issue:`160`).
 
     >>> create_sequential('year=2016..2019')
     year | 2016 | 2017 | 2018 | 2019
          |    0 |    1 |    2 |    3
 
-* inverted `name` and `labels` arguments when creating an Axis.
-  Argument `name` is optional (anonymous axis).
-  It is also possible to create an Axis by passing a single string of the kind 'name=labels':
+* replaced *args, **kwargs by explicit arguments in documentation of aggregation functions (sum, prod, mean, std,
+  var, ...). Closes :issue:`41`.
 
-    >>> anonymous = Axis('0..100')
-    >>> age = Axis('age=0..100')
-    >>> gender = Axis('M,F', 'gender')
+* improved documentation of plot method (closes :issue:`169`).
+
+* improved auto-completion in interactive consoles for both LArray and Session objects. LArray objects can now
+  complete keys within [].
+
```

Comment:
My question was not about `tab` but about to specify `using tab` after `within []`.